Configurations/common.tmpl

```diff
@@ -211,6 +219,34 @@
      $cache{$dir} = 1;
  }
 
+ # do{html,man}docs are responsible for building documentation from .pods.
+ # They will call generatesrc.
+ sub dohtmldocs {
+     my $section = shift;
+     foreach my $doc (@{$unified_info{htmldocs}->{$section}}) {
+         next if $cache{$doc};
+         $OUT .= generatesrc(src => $doc,
+                             generator => $unified_info{generate}->{$doc});
+         foreach ((@{$unified_info{depends}->{$doc}})) {
+             dogenerate($_, undef, undef, %opts);
+         }
+         $cache{$doc} = 1;
+     }
+ }
+
+ sub domandocs {
+     my $section = shift;
+     foreach my $doc (@{$unified_info{mandocs}->{$section}}) {
+         next if $cache{$doc};
+         $OUT .= generatesrc(src => $doc,
+                             generator => $unified_info{generate}->{$doc});
+         foreach ((@{$unified_info{depends}->{$doc}})) {
+             dogenerate($_, undef, undef, %opts);
+         }
+         $cache{$doc} = 1;
+     }
+ }
```

Comment:
These two functions seem virtually identical. Can we not merge them somehow?

Reply:
Sure